src/components/Plugin/CommentBox/Comment.vue

```diff
@@ -18,14 +18,13 @@ const props = defineProps({
   mainThread: String,
   space: Object
 });
-const { web3 } = useWeb3();
+const { web3Account } = useWeb3();
```

Comment:
We can remove the computed import

Reply:
This applies to some other files too

Reply:
Yep. I think we should really add the linting ass a commit hook or so.